lib/Fetcher/FeedFetcher.php

```diff
@@ -344,6 +344,30 @@ protected function buildItem(
         return $item;
     }
 
+    /**
+     * Helper function for fetching favicon urls
+     *
+     * @param string        $base_url  Base URL for the feed
+     *
+     * @return string|mixed|bool
```

Comment:
When would it be `mixed`?

Reply:
That function is actually only `string|bool`
https://github.com/ArthurHoaro/favicon/blob/master/src/Favicon/Favicon.php#L142-L144